pkg/execution/state/redis_state/backlog.go

```diff
@@ -273,8 +275,9 @@ type BacklogThrottle struct {
 }
 
 type QueueBacklog struct {
-	BacklogID         string `json:"id,omitempty"`
-	ShadowPartitionID string `json:"sid,omitempty"`
+	BacklogID               string `json:"id,omitempty"`
+	ShadowPartitionID       string `json:"sid,omitempty"`
+	EarliestFunctionVersion int    `json:"fv,omitempty"`
```

Comment:
what's the reason for calling it `Earliest`?
it gives an impression that the version number should be low.